Library/Homebrew/formulary.rb

```diff
@@ -198,12 +199,14 @@ def self.path(ref)
     loader_for(ref).path
   end
 
-  def self.loader_for(ref)
+  def self.loader_for(ref, is_installing = false, warn_all_ambiguity = false)
```

Comment:
This should instead take an `options` hash and use the last two parameter names as hash keys. This will improve readability of callers.